enterprise/frontend/src/metabase-enterprise/content_translation/components/ContentTranslationConfiguration.tsx

```diff
@@ -87,6 +100,16 @@ const UploadForm = ({
   const handleFileChange = async (event: ChangeEvent<HTMLInputElement>) => {
     if (event.target.files?.length) {
       const file = event.target.files[0];
+
+      if (file.size > maxContentDictionarySizeInBytes) {
+        setErrorMessages([
+          c("{0} is a number")
+            .t`Upload a dictionary smaller than ${approxMaxContentDictionarySizeInMB} MB`,
```

Comment:
![Screenshot 2025-05-28 at 2 17 22 PM](https://github.com/user-attachments/assets/e29a3edd-4a0c-417d-a9e9-413eb80d8c6a)

I think the way this is displayed in context it should probably say something like "Dictionary should be less than 1.5 MB"